patches/disable-secure-storage/matrix-react-sdk+3.57.0.patch

```diff
@@ -0,0 +1,22 @@
+diff --git a/node_modules/matrix-react-sdk/src/Lifecycle.ts b/node_modules/matrix-react-sdk/src/Lifecycle.ts
+index 1e7fae8..748fa29 100644
+--- a/node_modules/matrix-react-sdk/src/Lifecycle.ts
++++ b/node_modules/matrix-react-sdk/src/Lifecycle.ts
+@@ -62,6 +62,7 @@ import { DialogOpener } from "./utils/DialogOpener";
+ import { Action } from "./dispatcher/actions";
+ import AbstractLocalStorageSettingsHandler from "./settings/handlers/AbstractLocalStorageSettingsHandler";
+ import { OverwriteLoginPayload } from "./dispatcher/payloads/OverwriteLoginPayload";
++import TchapUserSettings from '../../../src/util/TchapUserSettings';
+ 
+ const HOMESERVER_URL_KEY = "mx_hs_url";
+ const ID_SERVER_URL_KEY = "mx_is_url";
+@@ -819,6 +820,9 @@ async function startMatrixClient(startSyncing = true): Promise<void> {
+         // before the client.
+         await EventIndexPeg.init();
+         await MatrixClientPeg.start();
++        //:tchap: override user settings after the client init
```

Comment:
Is that possible to listen to the dispatcher instead of using a patch ?
I see 2 events sent in this function : ` dis.dispatch({ action: 'will_start_client' }, true);` or `dis.dispatch({ action: 'client_started' });`

Reply:
good point, I'll give it a try